### ISSUE TITLE
TypeError: 'NormalizedLandmarkList' object is not iterable In pose model 

### ISSUE THREAD

**Issue by Woong0416** [google/mediapipe#3783]
I need some help in my code .. the error is 

"TypeError: 'NormalizedLandmarkList' object is not iterable"

my code is

```  import cv2
import mediapipe as mp
import numpy as np
import math

cap= cv2.VideoCapture(0)

mp_drawing = mp.solutions.drawing_utils
mp_pose = mp.solutions.pose

pose = mp_pose.Pose(
    min_detection_confidence=0.5,
    min_tracking_confidence=0.5)

imgWidth = cap.get(cv2.CAP_PROP_FRAME_WIDTH)
imgHeight = cap.get(cv2.CAP_PROP_FRAME_HEIGHT)

while cap.isOpened():
    success, img = cap.read()
    if not success:
      print("Ignoring empty camera frame.")
      # If loading a video, use 'break' instead of 'continue'.
      continue

    imgae = cv2.flip(img,1)
    img = cv2.cvtColor(img, cv2.COLOR_BGR2RGB)

    results = pose.process(img)

    img = cv2.cvtColor(img, cv2.COLOR_RGB2BGR)

    if results.pose_landmarks is not None:
        for res in results.pose_landmarks:
            joint = np.zeros((50, 3))
            for j, lm in enumerate(res.landmark):
                joint[j] = [lm.x, lm.y, lm.z]

            #no.33
            joint[33,0] = joint[0,0]
            joint[33,1] = joint[0,1]
            joint[33,2] = joint[0,2]
            joint33_x = joint[33,0]
            joint33_y = joint[33,1]
    
            pixel21 = mp_drawing._normalized_to_pixel_coordinates(joint33_x, joint33_y, imgWidth, imgHeight)


            cv2.circle(img, pixel21, 3, (0, 255, 0), -1)


    # Flip the image horizontally for a selfie-view display.
    cv2.imshow('MediaPipe Pose', img)
    if cv2.waitKey(5) & 0xFF == 27:
        break ```

**Comment by kuaashish** [comment]
Hi @Woong0416, 
Have you looked at this closed issue #3191 and [#3191(comment)](https://github.com/google/mediapipe/issues/3191#issuecomment-1091095268). Thank you!

**Comment by kuaashish** [comment]
Hi @Woong0416,
Please go through the above comment and let us know if the error still persists or has been resolved. Thank you!
